scio-grpc/src/main/scala/com/spotify/scio/grpc/SCollectionSyntax.scala

```diff
@@ -92,6 +92,65 @@ class GrpcSCollectionOps[Request](private val self: SCollection[Request]) extend
       .mapValues(_.asScala.map(_.asScala))
   }
 
+  def batchGrpcLookup[
+    BatchRequest: Coder,
```

Comment:
I don;t think we need this coder since the type is internal to the transform

Reply:
Makes sense .. fixed